alignCom.html

```diff
@@ -0,0 +1,67 @@
+//This<!DOCTYPE html>
+<html>
+<head>
+  <title>Align Commerce JS Test</title>
+
+  <style>
+  div{
+    margin-top: 30%;
+    position: relative;
+    bottom: 45%;
+    left: 35%; 
+    font-size: 3rem;
+  }
+
+  </style>
+</head>
+	<body>
+    <div>
+      Backgrond color changed for: <span id="text"></span>
+    </div>
+  </body>
+    
+</html>
+
+<script type="text/javascript">
+	(function(){
+		'use strict'
+		var body = document.body;
+    var text = document.getElementById('text');
+		var usedColor = {};
+		
+		var timer = 1;
+
+    var getRandomNum = function(){
+      return Math.round(Math.random() * 255);
+    }
+
+    var formNumArr = function(){
+      return [getRandomNum(), getRandomNum(), getRandomNum()];
+    }
+
+    var getColor = function(){
+      var numArr = formNumArr();
+
+      if (usedColor[numArr]){
+        getColor();
+      } else {
+        usedColor[numArr] = true;
+        return 'rgb(' + numArr[0] + ',' + numArr[1] + ',' + numArr[2] + ')';
+      }
+
+    }
+
+		var changeColor = setInterval(function () {
+      var color = getColor();
+
+      body.style.backgroundColor = getColor();
+      text.innerHTML = timer === 1 ? timer + ' time' : timer + ' times';
+
+		   if (timer === 10) {
```

Comment:
This doesent seem too flexible, what if we want to use fractions (10.55) seconds